### ISSUE TITLE
fairness issues for comparision

### ISSUE THREAD

**Issue by amsword** [HobbitLong/SupContrast#27]
I find the proposed training strategy is 1) train the backbone with the labels and the contrastive loss, 2) finetune the last linear layer. The baseline approach is train the backbone and last linear layer at the same time with cross entropy loss. Do you have a reference of 1) train the backbone with cross entropy loss, 2) re-train the last linear layer from scratch? 

The reason is that, the baseline here has multiple differences with the proposed solution. The gain could come from more iterations, i.e. iterations in pre-training + iterations in fine-tuning.

**Comment by HobbitLong** [comment]
@amsword, excellent question

IIRC, the baseline you proposed here is sub-optimal compared to end-to-end cross-entropy training. In theory it also should be, but you can still run such a baseline to verify.

**Comment by Chen-Song** [comment]
If cross-entropy and SCL(supervised contrastive learning) are regarded as a multi-task task, which means that the network is in the form of a backbone and multiple branches, what will be the result? Could it be better? This also avoids finetuning the last linear layer.
